CONTRIBUTING.md

```diff
@@ -29,20 +29,22 @@ The following text equips you with knowledge that makes contributing to ilastik
    gh repo clone ilastik
    ```
 
-1. If you have an existing [Miniconda][miniconda] installation, you can use it for ilastik development, but install [Mamba][mamba] and configure conda to use [strict channel priority][strict-channel-priority]:
+1. If you have an existing [Miniconda][miniconda] installation, you can use it for ilastik development, but configure conda to use [strict channel priority][strict-channel-priority]. If your conda version is older than 23.10.0, make sure install `mamba`, and for all following steps, run the commands with "`mamba`" instead of "`conda`" (or consider updating conda instead):
```

Comment:
And I'm not sure about the mixing of miniconda, miniforge. I would ditch all mention of miniconda.

I wouldn't suggest that. I don't think there are guarantees for older versions of conda to keep up with the ecosystem. I think we _can_ be opinionated here.

```suggestion
1. If you have an existing [Miniconda][miniconda] installation, you can use it for ilastik development, but configure conda to use [strict channel priority][strict-channel-priority].
```